eclipse-collections-code-generator/src/main/resources/api/map/primitivePrimitiveMap.stg

```diff
@@ -81,6 +82,21 @@ public interface <name1><name2>Map extends <name2>ValuesMap
      */
     void forEachKeyValue(<name1><name2>Procedure procedure);
 
+    /**
+     * Implements the injectInto pattern with each key/value pair of the map.
+     * @param value to be injected into the map
+     * @param function to apply to the injected value and key/value pairs
+     * @return result of injecting the injectedValue into the map
+     * @since 11.1
+     */
+    default \<IV> IV injectIntoKeyValue(IV injectedValue,
+    Object<name1><name2>ToObjectFunction\<? super IV, ? extends IV> function)
+    {
+        IV[] result = (IV[]) new Object[]{injectedValue};
+        this.forEachKeyValue((key, value) -> result[0] = function.valueOf(result[0], key, value));
+        return result[0];
+    };
```

Comment:
There is an unnecessary semi-colon after the closing curly brace for the method declaration.